site_root/source/resources/StructureDefinition-Participant.json

```diff
@@ -0,0 +1,26 @@
+{
+  "resourceType": "StructureDefinition",
+  "url": "http://fhir.kids-first.io/StructureDefinition/Participant",
+  "name": "Participant",
+  "status": "draft",
+  "fhirVersion": "4.0.0",
+  "kind": "resource",
+  "abstract": false,
+  "type": "Patient",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Patient",
+  "derivation": "constraint",
+  "differential": {
+    "element": [
+      {
+        "id": "Patient.gender",
+        "path": "Patient.gender",
+        "min": 1
```

Comment:
why?

Reply:
Huh, I didn't catch this. Yeah, gender seems not to be required. Maybe:

```json
{
  "id": "Patient.gender",
  "path": "Patient.gender",
  "min": 0,
  "max": 1,
}
```

Reply:
It was for testing purposes so I could test an invalid participant that didn’t have gender. I could remove it if you really want me to but this whole SD will get overwritten when Meen does the Participant SD anyway